src/pages/ForContractors.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import NavBar from '../components/NavBar';
+import '../styles/NavBar.css';
+
+//TODO: more to add here. do not change to implicit return
+const ForContractors = () => {
+    return (
+        <>
```

Comment:
yuo need a fragment here

Reply:
On all of these i will need the fragment, just like i will need to brackets and return statement. there is much more to add to these.